docs/changes/coordinates/11690.other.rst

```diff
@@ -0,0 +1 @@
+SkyCoord links now use intersphinx links to to work in affiliate packages.
```

Comment:
Actually, maybe do expand that it now works if `SkyCoord` is subclassed (which I think is problematic for other reasons...)

Reply:
Done!

"SkyCoord links now use intersphinx links for affiliate package convenience.
Outside the astropy package, subclasses of SkyCoord, with inherited docstrings,
will now link to the astropy docs instead of raising a warning / error."